bedms/attr_standardizer.py

```diff
@@ -79,18 +93,54 @@ def _get_parameters(self) -> Tuple[int, int, int, int, int, float]:
                 OUTPUT_SIZE_BEDBASE,
                 DROPOUT_PROB,
             )
-        else:
-            raise ValueError(
-                f"Schema not available: {self.schema}. Presently, three schemas are available: ENCODE , FAIRTRACKS, BEDBASE"
+    
+        raise ValueError(
+                f"Schema not available: {self.schema}."
+                "Presently, three schemas are available: ENCODE , FAIRTRACKS, BEDBASE"
             )
 
-    def _load_model(self) -> nn.Module:
+    def _load_model(self) -> tuple[nn.Module, object, object]:
```

Comment:
Is this syntax (`tuple[...]`) supported? Do we not need `typing.Tuple`?

Reply:
yeah, we need

Reply:
correct me if I am wrong here... in my mind `tuple` is ok for python >= 3.10

`typing.Tuple` is needed for anything less?